apps/codeforafrica/src/components/SocialMediaConnectButton/SocialMediaConnectButton.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-env browser */
+import SvgIcon from "@mui/material/SvgIcon";
+import React from "react";
+
+import FacebookIcon from "@/codeforafrica/assets/icons/Type=facebook, Size=24, Color=CurrentColor.svg";
+import GithubIcon from "@/codeforafrica/assets/icons/Type=github, Size=24, Color=CurrentColor.svg";
+import InstagramIcon from "@/codeforafrica/assets/icons/Type=instagram, Size=24, Color=CurrentColor.svg";
+import LinkedinIcon from "@/codeforafrica/assets/icons/Type=linkedin, Size=24, Color=CurrentColor.svg";
+import SlackIcon from "@/codeforafrica/assets/icons/Type=slack, Size=24, Color=CurrentColor.svg";
+import TwitterIcon from "@/codeforafrica/assets/icons/Type=twitter, Size=24, Color=CurrentColor.svg";
+import SocialMediaButton from "@/codeforafrica/components/SocialMediaButton";
+
+function getButtonIcon(name) {
```

Comment:
```suggestion
function getIcon(name) {
```

... and instead of method, we can also just use an object. e.g.

```js
const ICON_BY_NAME = {
  facebook: FacebookIcon,
}
```

And get the icon as:
```js
  ICON_BY_NAME[name.toLowerCase()];
```